### PR TITLE
Us15

### PR DESCRIPTION
**Pull request by Unikorn44**
New PR, with dev this time

### CODE REVIEW

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You need to merge with dev and get what Ludovic sent into your branch

File: public/assets/css/style.css [hunk 1]
```diff
@@ -8,7 +8,11 @@
     justify-content: space-around;
 
     position:fixed;
+<<<<<<< HEAD
+    z-index:9999;
+=======
     z-index:10;
+>>>>>>> dev
```
Comment (aldaitzwild) [inline public/assets/css/style.css:15]: You have a conflict !

File: src/View/SoughtProfile/soughtProfile.html.twig [hunk 2]
```diff
@@ -51,13 +51,21 @@
...
                 }
                 if(Math.abs(between) >= distance && directionCursor == "right" && profileCard.getAttribute("class") != "rightMove") {
                     profileCard.setAttribute("class", "rightMove");
                     setTimeout(() => {
+<<<<<<< HEAD
+                        document.location.href="/../../soughtProfile/matchUpdate/"+searchingId+"/"+profileId+"/accept";
+=======
                         document.location.href="/soughtProfile/matchUpdate/"+searchingId+"/"+profileId+"/accept";
+>>>>>>> dev
                     }, 1000);
```
Comment (aldaitzwild) [inline src/View/SoughtProfile/soughtProfile.html.twig:69]: Yeah two more conflicts !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/SoughtProfile/soughtProfile.html.twig (hunk 2 above)
Comment (Unikorn44) [reply src/View/SoughtProfile/soughtProfile.html.twig]: Conflicts resolved

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Quality control failed : 
FILE: src/Controller/ConnectController.php
----------------------------------------------------------------------
FOUND 21 ERRORS AND 1 WARNING AFFECTING 21 LINES
----------------------------------------------------------------------
  1 | WARNING | [ ] A file should declare new symbols (classes,
    |         |     functions, constants, etc.) and cause no other
    |         |     side effects, or it should execute logic with
    |         |     side effects, but should not do both. The first
    |         |     symbol is defined on line 8 and the first side
    |         |     effect is on line 5.
  3 | ERROR   | [x] Header blocks must be separated by a single blank
    |         |     line
  4 | ERROR   | [x] Header blocks must be separated by a single blank
    |         |     line
 17 | ERROR   | [x] Expected 1 space(s) after closing parenthesis;
    |         |     found " /*vérification remplissage champs*/\n   
    |         |         "
 20 | ERROR   | [x] Space before opening parenthesis of function call
    |         |     prohibited
 21 | ERROR   | [x] Space before opening parenthesis of function call
    |         |     prohibited
 22 | ERROR   | [x] Whitespace found at end of line
 24 | ERROR   | [x] Whitespace found at end of line
 25 | ERROR   | [x] Whitespace found at end of line
 27 | ERROR   | [x] Expected 1 space(s) after IF keyword; 0 found
 27 | ERROR   | [x] Expected 1 space(s) after closing parenthesis;
    |         |     found newline
 29 | ERROR   | [x] Whitespace found at end of line
 30 | ERROR   | [x] Expected 1 space after closing brace; newline
    |         |     found
 31 | ERROR   | [x] Expected 1 space(s) after ELSE keyword; newline
    |         |     found
 33 | ERROR   | [x] Expected 1 space(s) after closing parenthesis;
    |         |     found " /*comparaison des passwords*/\n          
    |         |          "
 34 | ERROR   | [x] Whitespace found at end of line
 36 | ERROR   | [x] Whitespace found at end of line
 38 | ERROR   | [x] Expected 1 space after closing brace; newline
    |         |     found
 39 | ERROR   | [x] Expected 1 space(s) after ELSE keyword; newline
    |         |     found
 43 | ERROR   | [x] Whitespace found at end of line
 46 | ERROR   | [x] Function closing brace must go on the next line
    |         |     following the body; found 1 blank lines before
    |         |     brace
 47 | ERROR   | [x] Expected 1 blank line at end of file; 2 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 21 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: src/Model/ConnectManager.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 37 | WARNING | Line exceeds 120 characters; contains 126 characters
----------------------------------------------------------------------

**Comment by Unikorn44** [comment]
Bugs corrected, code cleaned

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Code quality : 

FILE: src/Model/ConnectManager.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 37 | WARNING | Line exceeds 120 characters; contains 126 characters
----------------------------------------------------------------------

**Comment by Unikorn44** [comment]
Corrected some lines to pass the quality test

**Pull request closed without merging**